obs2ioda-v2/README.md

```diff
@@ -54,9 +73,34 @@ To install the NCEP BUFR library, follow these steps:
    ```
 5. To locate the NCEP BUFR library, run:
    ```bash
-   find . -name *libbufr*
+   bash -c "find . -name *libbufr*"
```

Comment:
Is there a specific reason this `find` command needs to be run in the bash shell?